### DISCUSSION TITLE
Very low Scroll senstivity in Dolphin File manager.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [prasanthrangan/hyprdots#194] [Q&A] [answered]
I created a new guest user , and with default config the senstivity of dolphin there is perfect. How do i fix this in my actual setup ?

**Comment by prasanthrangan** [comment]
hmmm, are you using mouse or touchpad gestures to scroll? and is it just for dolphin?

**Reply by KAGEYAM4** [reply]
The touchpad is only a little bit faster but otherwise same experience as mouse. Only dolphin is affected by this, the scroll ( touchpad + mouse ) works fine in other softwares like firefox, freetube, kitty.

**Reply by prasanthrangan** [reply]
hmm, so if the scroll works fine with default hyprland config, can you try enabling/disabling the QT env variables in hyprland.conf?

**Reply by KAGEYAM4** [reply]
`env = QT_QPA_PLATFORMTHEME,qt5ct`  this is creating problem , but if disabled no theme is applied.

also `env = QT_STYLE_OVERRIDE,kvantum` this is already disabled , should i enable it ?

**Reply by KAGEYAM4** [reply]
@prasanthrangan  any update ?

**Reply by prasanthrangan** [reply]
hmm, unfortunately `env = QT_QPA_PLATFORMTHEME,qt5ct` is required for the kvantum themes to work.
you can try `env = QT_STYLE_OVERRIDE,kvantum` but it will not apply the colors to QT apps so it will break the look.
Not sure if there is any other way to fix scroll speed without disabling `env = QT_QPA_PLATFORMTHEME,qt5ct` , but you can try raising this to hyprland team

**Reply by KAGEYAM4** [reply, marked as answer]
![image](https://github.com/prasanthrangan/hyprdots/assets/75798544/53c5f723-3378-4100-b5c9-302f4be600a7)
It was a setting under `qt5ct` , can you check what is scroll value for you ?

**Reply by prasanthrangan** [reply]
It was set to 1 by default, changing this does fix the scroll speed!

**Reply by prasanthrangan** [reply]
I'll updated the qt5ct config and set 5 as default!

**Reply by KAGEYAM4** [reply]
5 was too fast for me, i switched to `3`. 

Also , maybe add `troubleshooting` section in readme, where you can mention this setting so people can tweak it themselves if they ran into this problem.

**Reply by prasanthrangan** [reply]
yup, makes sense.. will do!
